src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link should refer to the compiled CSS file, not the SCSS source file. Change the `href` attribute to point to the correct `.css` file.